schemas/json/theme.json

```diff
@@ -226,6 +226,60 @@
 								"type": "string"
 							},
 							"default": [ "px", "em", "rem", "vh", "vw", "%" ]
+						},
+						"customSpacingSize": {
+							"description": "Allow users to set custom space sizes.",
+							"type": "boolean",
+							"default": true
+						},
+						"spacingSizes": {
+							"description": "Space size presets for the space size selector.\nGenerates a single class (`.has-{slug}-space-size`) and custom property (`--wp--preset--space-size--{slug}`) per preset value.",
```

Comment:
This no longer generates a classname like `.has-{slug}-space-size` — shall we update the description?

Reply:
or should we generate the classnames?

Reply:
I'd lean toward avoiding generating classnames for the moment, until we have a better idea of how they'd be used. Since spacing will be used for `blockGap`, which has a variety of different outputs and doesn't map neatly to a single property (it's sometimes used for `gap` and sometimes used for `margin-top`), it might be better to skip it for now?

Reply:
Ok, have removed, lets circle back on classnames laters

Reply:
Thanks, that sounds good. I'm hopeful that if https://github.com/WordPress/gutenberg/pull/40875 (using layout type classnames to support block-level gap in theme.json) winds up being viable, in a follow-up, we could take a more detailed look at the classnames we're outputting.